src/ucp/proto/proto_common.c

```diff
@@ -720,28 +720,28 @@ ucs_status_t ucp_proto_request_init(ucp_request_t *req)
 }
 
 /**
- * Current implementation of @ref ucp_proto_t::clean supports the only case if
- * no any data was sent.
+ * Current implementation of @ref ucp_proto_t::reset supports the only case if
```

Comment:
Current implementation of @ref ucp_proto_t::reset supports only the case that no data was sent yet.